tensorflow/python/ops/math_ops.py

```diff
@@ -340,6 +340,25 @@ def multiply(x, y, name=None):  # pylint: disable=missing-docstring
   # Do an is comparison here since this is cheaper than isinstance or __eq__
   if y is 1:  # pylint: disable=literal-comparison
     return x
+  """Returns 'x' * 'y' element wise.
+  
+  For example:
+  
+  >>> x = 32
+  >>> y = 34
+  >>> tf.multiply(x, y)
+  <tf.Tensor: shape=() dtype=int32>
+  #returns 1088
```

Comment:
It looks like the output was computed in tf1, as otherwise the tensor would have a value and you wouldn't need the comment.

Recompute?

Reply:
Got it, I'll take out the comments and resubmit

Reply:
Made the changes as per your guidance